CMakeLists.txt

```diff
@@ -249,6 +249,7 @@ set_target_properties(kqueue PROPERTIES SOVERSION 0)
 if(WIN32)
   target_compile_definitions(kqueue PRIVATE _USRDLL;_WINDLL)
   target_compile_definitions(kqueue PRIVATE _CRT_SECURE_NO_WARNINGS)
+  target_compile_definitions(kqueue PRIVATE WIN32_LEAN_AND_MEAN)
```

Comment:
the `windows.h` include in debug brought in the whole kitchen sink so  we define this globally as to not redefine any structs